.circleci/config.yml

```diff
@@ -1,31 +1,59 @@
-# Javascript Node CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-javascript/ for more details
-#
-version: 2
-jobs:
-  build:
-    docker:
-      - image: circleci/node
+defaults: &defaults
+  working_directory: /tmp/marksy
+  docker:
+    - image: circleci/node:8
 
-    working_directory: ~/repo
+version: 2
 
+jobs:
+  size:
+    <<: *defaults
     steps:
       - checkout
-
-      # Download and cache dependencies
       - restore_cache:
           keys:
-          - v1-dependencies-{{ checksum "package-lock.json" }}
-          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-{{ checksum "yarn.lock" }}
           - v1-dependencies-
+      - run: yarn install
+      - save_cache:
+          paths:
+            - node_modules
+          key: v1-dependencies-{{ checksum "yarn.lock" }}
+      - run: yarn size
```

Comment:
Let's remove `size-limit` from `test` npm script then?

Reply:
I did, didn't I?

Reply:
No it's still there https://github.com/storybooks/marksy/blob/master/package.json#L27